test/test_operations.py

```diff
@@ -2711,18 +2711,6 @@ def test_dlpack_pytorch_cuda_to_xla(self):
     t2_cuda.fill_(6)
     self.assertTrue(torch.allclose(xla_t2.cpu(), t2_cuda.cpu()))
 
-    cuda1 = torch.device('cuda:1')
```

Comment:
this test shouldn't be removed. Neither should the one below.

Reply:
Ack. I removed it as the new runner only had 1 GPU but now it is reverted. I will add the triton tests separately.